roles/installer/templates/deployment.yaml.j2

```diff
@@ -111,6 +161,17 @@ spec:
           args: {{ web_args }}
 {% endif %}
           imagePullPolicy: '{{ image_pull_policy }}'
+          livenessProbe:
+            httpGet:
+              path: /api/v2/ping/
+              port: 8052
+            initialDelaySeconds: 90
+            timeoutSeconds: 2
+          readinessProbe:
+            httpGet:
+              path: /api/v2/ping/
+              port: 8052
+            timeoutSeconds: 2
```

Comment:
shouldn't it be configured from outside with predefined default values?

Reply:
#1188 - something like that
that way the user will have more control over the deployment if needed

Reply:
@erz4 
I agree - it just was not the aim for my PR.
I wanted this PR to be a basic minimum to add this functionality ASAP, as every change to the deployment causes downtimes & job failures.

But since this PR is still open after now nearly 8 months, that obviously did not go as desired and the problem still persists.

If your PR is merged, this can be closed. We just need these checks.

Reply:
@moonrail 
did you test it locally with molecule?

my probes failing in the docker-desktop env but working on a k8s cluster

Reply:
I've tested with molecule but also in our test environment (k8s).
Both ran at the last try (several months ago).

Reply:
Yup mine is failing in the migration phase as mentioned above.

So what is the status now? why is that PR stuck?